src/util/DebouncedOnChange/index.tsx

```diff
@@ -60,7 +60,7 @@ export default class DebouncedOnChangeComponent extends React.Component<
       this.props.onChange(newValue);
     }
 
-    if (this.props.debouncedOnChange) {
+    if (this.props.debouncedOnChange && this.debouncedOnChange) {
```

Comment:
Why is this one necessary if you're initializing it in the constructor?

Reply:
It's only initialized if props.debouncedOnChange is true. Typescript doesn't realize that props.debouncedOnChange being true is equivalent to this.debounceOnChanged being defined, so it complains that it may be undefined.